generators/server/templates/src/main/java/package/domain/_Authority.java

```diff
@@ -29,6 +29,13 @@
     @Column(length = 50)<% } %>
     private String name;
 
+    public Authority() {
```

Comment:
If you remove this one then the empty constructor can also be removed.
I added this constructor for convenience but of course null values should be handled.
Note that when you call `new Authority()`, name is also not set as there is no default value for it so it's not really better...